help/global-feed.md

```diff
@@ -1,12 +1,12 @@
-# All messages
+# Global feed
 
-{!all-messages.md!}
+{!global-feed.md!}
 
 !!! keyboard_tip ""
 
-    Use <kbd>S</kbd> (go to stream) or <kbd>Shift</kbd> +
-    <kbd>S</kbd> (go to conversation) to zoom in, and <kbd>A</kbd> to
-    get back to **All messages**.
+    Use <kbd>S</kbd> (narrow to stream) or <kbd>Shift</kbd> +
+    <kbd>S</kbd> (narrow to conversation) to zoom in, and <kbd>A</kbd> to
```

Comment:
Please be more careful when rebasing; you appear to have accidentally reverted part of ac47f85a34a1abb79d361678992eefcb51d4d3be. Can you audit this PR for similar mistakes and correct them?